cli/cook/subcommands/show.py

```diff
@@ -46,6 +46,12 @@ def tabulate_job(cluster_name, job):
                       ['Priority', job['priority']]]
     if job['max_runtime'] != DEFAULT_MAX_RUNTIME:
         job_definition.append(['Max Runtime', millis_to_timedelta(job['max_runtime'])])
+    if 'disk' in job:
+        job_definition.append(['Disk Request', job['disk']['request']])
+        if 'limit' in job['disk']:
+            job_definition.append(['Disk Limit', job['disk']['limit']])
+        if 'type' in job['disk']:
+            job_definition.append(['Disk Type', job['disk']['type']])
```

Comment:
Could you share an example of what this looks like through `cs show`?

Reply:
<img width="1033" alt="Screen Shot 2020-11-11 at 11 51 12 AM" src="https://user-images.githubusercontent.com/52860192/98840244-8db00100-2414-11eb-84a4-4473aa22b182.png">
<img width="1005" alt="Screen Shot 2020-11-11 at 11 52 27 AM" src="https://user-images.githubusercontent.com/52860192/98840245-8e489780-2414-11eb-9e1f-ab68c8ebe4bd.png">
<img width="1013" alt="Screen Shot 2020-11-11 at 11 53 18 AM" src="https://user-images.githubusercontent.com/52860192/98840250-8f79c480-2414-11eb-8da1-582fa18786bf.png">

Reply:
Discussed offline moving GPUs and Disk rows to right under Memory and CPUs

Reply:
moved GPUs before Disk rows but leaving the required fields above the optional fields